docker-compose.staging.yaml

```diff
@@ -4,8 +4,13 @@ services:
 
   app:
     image: usedgoodreads/app:staging
-    ports:
-      - "8080:5000"
 
   worker:
     image: usedgoodreads/app:staging
+
+  nginx:
+    volumes:
+      - /etc/letsencrypt/live/usedgoodreads.com/:/certs
+    ports:
+      - 8080:80
+      - 4443:443
```

Comment:
Same here

Reply:
Fixed in cedc53afb99cdda9d67c86db6ac8f7ebb2c7c887